src/Test/Utilities/Portable/Compilation/CompilationExtensions.cs

```diff
@@ -309,6 +309,21 @@ public static void ValidateIOperations(Func<Compilation> createCompilation)
                             Assert.True(argument.Descendants().All(n => n.IsImplicit), $"Explicit node in default argument value ({argument.Syntax.RawKind}): {argument.Syntax.ToString()}");
                         }
                     }
+
+                    // Make sure that all static member references or invocations of static methods do not have implicit IInstanceReferenceOperations
+                    // as their receivers
+                    if (operation is IMemberReferenceOperation memberReference &&
+                        memberReference.Member.IsStatic &&
+                        memberReference.Instance is IInstanceReferenceOperation)
+                    {
+                        Assert.False(memberReference.Instance.IsImplicit, $"Implicit IInstanceReceiver on {operation.Syntax}");
```

Comment:
>IInstanceReceiver [](start = 85, length = 17)

What does this refer to?

Reply:
I'm not sure what you mean? If there is an implicit receiver, we assert.

---
In reply to: [149580256](https://github.com/dotnet/roslyn/pull/22972#discussion_r149580256) [](ancestors = 149580256)

Reply:
> I'm not sure what you mean? If there is an implicit receiver, we assert.

Is there an interface named IInstanceReceiver?

---
In reply to: [149832365](https://github.com/dotnet/roslyn/pull/22972#discussion_r149832365) [](ancestors = 149832365,149580256)

Reply:
Filed a follow up issue. https://github.com/dotnet/roslyn/issues/23090